corehq/ex-submodules/pillowtop/utils.py

```diff
@@ -145,15 +146,29 @@ def safe_force_seq_int(seq, default=None):
         return default
 
 
+def _get_consumer():
+    return KafkaConsumer(
+        client_id='pillowtop_utils',
+        bootstrap_servers=settings.KAFKA_BROKERS,
+        request_timeout_ms=1000
+    )
+
+
 def get_all_pillows_json():
     pillow_configs = get_all_pillow_configs()
-    return [get_pillow_json(pillow_config) for pillow_config in pillow_configs]
+    consumer = _get_consumer()
+    return [get_pillow_json(pillow_config, consumer) for pillow_config in pillow_configs]
 
 
-def get_pillow_json(pillow_config):
+def get_pillow_json(pillow_config, consumer=None):
     assert isinstance(pillow_config, PillowConfig)
 
+    def _is_kafka(checkpoint):
+        return checkpoint.sequence_format == 'json'
+
     pillow = pillow_config.get_instance()
+    if consumer is None:
```

Comment:
Probably could also put this into `_is_kafka`, but I'm not sure if it's even necessary. If it's not passed in, are you ever going to need it?